packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -274,8 +274,10 @@ metadata:
   labels:
 {{ include "common.standard.labels" .  | indent 4 }}
 data:
-  networkid: {{ .Values.genesis.networkId | quote }}
-  genesis.json: {{ .Values.genesis.genesisFileBase64 | b64dec | quote }}
+  networkid: {{ $.Values.genesis.networkId | quote }}
+{{- if eq (default $.Values.genesis.useGenesisFileBase64 "true") "false" }}
```

Comment:
IUUC this is saying that if `useGensisFileBase64 == false` then use the base 64 genesis file? That seems inverted from what it should be.